internal/service/project/data_source_projects_test.go

```diff
@@ -19,18 +19,18 @@ func TestAccProjectDSProjects_basic(t *testing.T) {
 		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
 	)
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckProjectTeamsIdsWithMinCount(t, 2) },
+		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckProjectTeamsIDsWithMinCount(t, 2) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccMongoDBAtlasProjectsConfigWithDS(projectName, orgID,
 					[]*admin.TeamRole{
 						{
-							TeamId:    conversion.StringPtr(acc.GetProjectTeamsIdsWithPos(0)),
+							TeamId:    conversion.StringPtr(acc.GetProjectTeamsIDsWithPos(0)),
```

Comment:
do you know why Ids didn't fail the linter before?

Reply:
I have a newer linter locally which picks these now, it was a small drive by I didn't bother in trying to switch to your version of the linter